File: service/src/main/java/hu/bsstudio/scores/qualification/QualificationScoreService.java
package hu.bsstudio.scores.qualification;

import hu.bsstudio.scores.qualification.model.QualifiedTeam;
import hu.bsstudio.team.model.DetailedTeam;
import reactor.core.publisher.Mono;

public interface QualificationScoreService {
    Mono<DetailedTeam> updateQualificationScore(final QualifiedTeam team);
}

File: server/service/src/main/kotlin/hu/bsstudio/robonaut/scores/qualification/QualificationScoreService.kt
package hu.bsstudio.robonaut.scores.qualification

import hu.bsstudio.robonaut.scores.qualification.model.QualifiedTeam
import hu.bsstudio.robonaut.team.model.DetailedTeam
import reactor.core.publisher.Mono

interface QualificationScoreService {
  fun updateQualificationScore(qualifiedTeam: QualifiedTeam): Mono<DetailedTeam>
}
